### DISCUSSION TITLE
Fonts

### DISCUSSION THREAD

**Discussion by Hayden-Badenhop** [CarolineChief/Tafe240261#7] [General]
Hi, I was wondering if anyone has added any different fonts as I would like to keep the style the same. Please let me know if you have or haven't. 

Thanks Hayden

**Comment by CarolineChiefari** [comment]
Hey,
Great idea keeping a uniformed font throughout the project.
I used Arial font for the 'Mortgage Calculator' title text block and Segoe UI for the calculations text blocks.
Thanks
Caroline

**Comment by NicholasGannon** [comment]
Hi all,

I'm currently working on a repository that provides a curated list of useful **font and typography** resources for designers/developers! I'd really appreciate if you could check it out and give it a star!

https://github.com/NicholasGannon/Font-Resources

Thank you
